lms/views.py

```diff
@@ -11,25 +13,75 @@ class CourseViewSet(viewsets.ModelViewSet):
     queryset = Course.objects.all()
     serializer_class = CourseSerializer
 
+    def perform_create(self, serializer):
+        course = serializer.save()
```

Comment:
Так ты делаешь два запроса - сначала `insert` на создание курса, потом `update` на установку владельца. Правильнее указать одной строкой:
```suggestion
        serializer.save(owner=self.request.user)
```
С уроком также